_posts/2023-01-19-opensearch-project-2022-recap-and-whats-next.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title:  "OpenSearch Project 2022 recap and what's next"
+authors:
+  - dtaivpp
+  - elifish
+date:   2023-01-19 00:00:00 -0700
+categories:
+  - technical-post
+meta_keywords: "OpenSearch, OpenSearch Dashboards, 2022, Contributors"
+meta_description: "OpenSearch's community has grown signifigantly during 2022. Take a look at what we are doing in 2023 to making building with OpenSearch better!"
```

Comment:
should read "significantly" and "to make building with"

Reply:
The keywords are a bit broad. I would suggest these edits: 
OpenSearch roadmap, OpenSearch dashboards, open-source solution for search and analytics, OpenSearch migrations

Reply:
@dtaivpp - Be sure to updatethe release date on your blog.